src/applications/static-pages/ivc-champva/10-7959f-1/App.js

```diff
@@ -26,14 +26,18 @@ const App = ({ formEnabled }) => {
         <h3>
           <strong>Option 1: Online</strong>
         </h3>
-        <a href="/health-care/foreign-medical-program/register-form-10-7959f-1">
+        <a
+          className="vads-c-action-link--green"
```

Comment:
The action link is available as a component now: https://design.va.gov/storybook/?path=/docs/components-va-link-action--docs